src/server/game/Handlers/ChatHandler.cpp

```diff
@@ -329,6 +329,11 @@ void WorldSession::HandleMessagechatOpcode(WorldPacket & recvData)
         }
     }
 
+    // prevent crash player
+    if (msg.find("| |Hquest") != std::string::npos) {
```

Comment:
Mm you don't even need the brackets :)

Reply:
I know, but I really prefer this code-style :P